server/config/config-env.js

```diff
@@ -18,7 +18,7 @@ const configEnv = {
             },
             aws: {
                 cdnUrl: 'https://cdn-test.kommunicate.io',
-                bucket: 'km-test-cdn',
+                bucket: 'cdn-kommunicate',
```

Comment:
is this change related to this issue?

Reply:
It needs to revert as we have migrated test cdn to new aws account

Reply:
@Ojashwa Please use bucket: 'km-test-cdn'

Reply:
> 
> 
> is this change related to this issue?

@priyadarshi sir could you please describe the changes?

Reply:
yes